misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -90,6 +92,29 @@ public static void generateDescriptor(String command) {
             throw new BalGenToolException("Process not successfully completed. Process is interrupted while" +
                     " running the protoC executor.", e);
         }
+        if (process.exitValue() != 0) {
+            BufferedReader br = null;
+            try {
+                br = new BufferedReader(new
+                        InputStreamReader(process.getErrorStream(), "UTF-8"));
+                String err;
+                String errMsg = "";
+                while ((err = br.readLine()) != null) {
+                    errMsg = errMsg.concat(System.lineSeparator().concat(err));
+                }
```

Comment:
Shall we use StringBuilder or + instead of string concat

Reply:
Fixed in d8298c4